lib/WebpackConfig.js

```diff
@@ -117,6 +118,16 @@ class WebpackConfig {
         this.manifestKeyPrefix = manifestKeyPrefix;
     }
 
+    configureManifestPlugin(manifestPluginOptionsCallback = () => {}) {
+        this.usePostCssLoader = true;
+
+        if (typeof manifestPluginOptionsCallback !== 'function') {
+            throw new Error('Argument 1 to enablePostCssLoader() must be a callback function.');
```

Comment:
`Argument 1 to configureManifestPlugin() must be a callback function.`

Maybe you could also add a test for that part (in `test/WebpackConfig.js`).